installers/windows/nsis/installer.nsi

```diff
@@ -17,16 +17,15 @@
 # along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
 
 !define /date MYTIMESTAMP "%y%m%d%H%M"

```

Comment:
using ``git log --format=%ct -1`` you can retrieve the committer date of the current git commit